applications/shape_detector/src/plane_detector.py

```diff
@@ -0,0 +1,939 @@
+#!/usr/bin/env python
+
+'''
+Multi planar plain detector
+==================
+
+Using depth image to compute depth planes locally.
+
+
+Usage:
+
+Environemt : 
+    C:\\Users\\udubin\\Documents\\Envs\\barcode
+
+Install : 
+
+
+
+'''
+
+import numpy as np
+import cv2 as cv
+import unittest
+from scipy.spatial.transform import Rotation as Rot
+#from scipy import interpolate 
+
+ # importing common Use modules 
+import sys 
+sys.path.append(r'..\Utils\src')
+from opencv_realsense_camera import RealSense
```

Comment:
what is opencv_realsense_camera - is it a known package, or something you built in your env?